endpoints/controllers/newusers.controller.js

```diff
@@ -0,0 +1,9 @@
+const { selectAllUsers } = require('../models/newusers.model');
+
+exports.getAllUsers = (req, res, next) => {
+    selectAllUsers()
+        .then(users => {
+            res.status(200).json(users);
+        })
```

Comment:
We should be using .send(...) which is where we should be sending our object with a key of users which holds the users array